Sources/DashX/DashXClient.swift

```diff
@@ -588,4 +607,15 @@ public class DashXClient {
     public func requestLocationPermission() {
         CLLocationManager().requestWhenInUseAuthorization()
     }
+    
+    // MARK: - Get ID of Notification
+    
+    public func getNotificationID(_ message: [AnyHashable : Any]) -> String? {
+        if let theJSONString = message["dashx"] as? String,
```

Comment:
use `Constants.DASHX_NOTIFICATION_DATA_KEY` instead of `"dashx"`

This method shouldn't be public

Reply:
Done.